src/testsuite/arith128_test_i64.c

```diff
@@ -195,6 +194,66 @@ db_vec_vsrad (vi64_t vra, vui64_t vrb)
 }
 #endif
 
+int
+test_vrld (void)
+{
+  vui64_t i1, i2, e;
+  vui64_t j;
+  int rc = 0;
+
+  printf ("\ntest_vrld Vector Rotate left doubleword\n");
+
+  i1 = (vui64_t) CONST_VINT64_DW (1, 2);
+  i2 = (vui64_t) CONST_VINT64_DW (2, 1);
+  e = (vui64_t) CONST_VINT64_DW (4, 4);
```

Comment:
obviously a nit, but one option for these repeated patterns is to line up the `=`.

Reply:
This is no functional change that should be handled is separate summit.